src/View/Incipit/incipit.html.twig

```diff
@@ -13,20 +13,22 @@
 {% endblock %}
 
 {% block content %}
-	<a class="logo" href="/"><img src="/assets/images/logo.svg"></a>
+	<main class="incipit d-md-inline-flex"></audioclass=""<audiocontrolsautoplayclass="audioPage"src="/assets/sound/turnpage.mp3">>
```

Comment:
Nope ! `</audioclass=""<audiocontrolsautoplayclass="audioPage"src="/assets/sound/turnpage.mp3">>` that's not valid HTML

Reply:
correction made on audio tag